libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/utils/RouteUtils.kt

```diff
@@ -0,0 +1,229 @@
+package org.maplibre.navigation.android.navigation.v5.utils
+
+import android.location.Location
+import org.maplibre.geojson.LineString
+import org.maplibre.geojson.Point
+import org.maplibre.navigation.android.navigation.v5.milestone.BannerInstructionMilestone
+import org.maplibre.navigation.android.navigation.v5.milestone.Milestone
+import org.maplibre.navigation.android.navigation.v5.models.BannerInstructions
+import org.maplibre.navigation.android.navigation.v5.models.DirectionsRoute
+import org.maplibre.navigation.android.navigation.v5.models.LegStep
+import org.maplibre.navigation.android.navigation.v5.models.VoiceInstructions
+import org.maplibre.navigation.android.navigation.v5.navigation.NavigationConstants
+import org.maplibre.navigation.android.navigation.v5.routeprogress.RouteProgress
+
+object RouteUtils {
```

Comment:
Would you mind adding the `findCurrentBannerText` function, please?

Reply:
Good to know! Seems unused for me, but if it used public I will add it again 👍

Reply:
Mhm, if it is unsued in the library, I am not sure if we should add it for third parties?

Reply:
Why not? We will provide some tools that make sense to use with our library. But sure it's for every function complicated to decide, if this is a good helper, or a function that should implemented by the developer itself.

Reply:
Yeah, I don't care about this specific function, we can include it or not. In general my feeling is, that we should rather try to keep unused helper functions minimal, if the library or the UI module don't use thzem, are we sure anyone else uses it? If only a minority of implementing projects are using a function, I am not sure it should be part of the core.

I guess there are some helpers that make total sense though, so I am not saying we should remove everything necessarily :)

Reply:
I have added this again for now ✅